### PR TITLE
 W-12138949 update titles headings to std bsj

### PR DESCRIPTION
**Pull request by bjoson-mulesoft**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 1]
```diff
@@ -1,7 +1,4 @@
-= Create a Monitor
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
+= Creating Monitors
```
Comment (hannanelson) [inline modules/ROOT/pages/afm-create-monitor.adoc:1]:
```suggestion
= Create Monitors
```

File: modules/ROOT/pages/afm-create-private-location.adoc [hunk 2]
```diff
@@ -1,7 +1,4 @@
-= Create a Private Location
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
+= Creating Private Locations
```
Comment (hannanelson) [inline modules/ROOT/pages/afm-create-private-location.adoc:1]:
```suggestion
= Create Private Locations
```

File: modules/ROOT/pages/afm-create-private-location.adoc [hunk 3]
```diff
@@ -35,14 +32,14 @@ To create a private location:
...
 The private location is added to the list of private locations. While API Functional Monitoring is registering the location, the status indicator for the location is yellow. After it is registered, if the environment in which the location was created is online, the location shows a green status indicator.
++
+. If the indicator is red: 
+.. Navigate to Runtime Manager and restart the environment. 
+.. Check the logs for the environment to troubleshoot the problem that caused it to go offline.
 
-If the indicator is red, navigate to Runtime Manager and restart the environment. Then, check the logs for the environment to troubleshoot the problem that caused it to go offline.
-
-== What to Do Next
+== What To Do Next
```
Comment (hannanelson) [inline modules/ROOT/pages/afm-create-private-location.adoc]:
```suggestion
== What to Do Next
```

Comment (hannanelson) [reply modules/ROOT/pages/afm-create-private-location.adoc]: per Chicago

File: modules/ROOT/pages/afm-delete-monitor.adoc [hunk 4]
```diff
@@ -1,7 +1,4 @@
-= Delete a Monitor
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
+= Deleting Monitors
```
Comment (hannanelson) [inline modules/ROOT/pages/afm-delete-monitor.adoc:1]:
```suggestion
= Delete Monitors
```

File: modules/ROOT/pages/afm-upload-monitor.adoc [hunk 5]
```diff
@@ -28,12 +25,12 @@ For details on file structure and file requirements, see xref:bat-example-test-s
...
+The *Monitor Detail* page opens. 
++
+. In the *Last executions* list, you can view the test result and how long the test ran.
 
-== Result
-
-The *Monitor Detail* page opens. In the *Last executions* list you can view the test result and how long the test ran.
-
-== What to do next
+== What To Do Next
```
Comment (hannanelson) [inline modules/ROOT/pages/afm-upload-monitor.adoc]:
```suggestion
== What to Do Next
```

File: modules/ROOT/pages/afm-view-test-history.adoc [hunk 6]
```diff
@@ -1,29 +1,22 @@
-= View Test History
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
-
-:imagesdir: ../assets/images
+= Viewing Test History
```
Comment (hannanelson) [inline modules/ROOT/pages/afm-view-test-history.adoc:1]: Viewing Test Histories?

File: modules/ROOT/pages/bat-reporting-task.adoc [hunk 7]
```diff
@@ -34,7 +34,7 @@ NOTE: You can configure a secret for the license key. If you do so, use the secr
 +
 * `DISABLED_FOR_CONFIGS`: Disables the reporter for one or more configurations. The value is the name of one configuration or multiple names that are separated by commas. Names do not include the `.dwl` file extension.
 
-=== What to Do Next
+=== What To Do Next
```
Comment (hannanelson) [inline modules/ROOT/pages/bat-reporting-task.adoc]:
```suggestion
=== What to Do Next
```

File: modules/ROOT/pages/bat-reporting-task.adoc [hunk 8]
```diff
@@ -71,15 +71,15 @@ NOTE: If you configured a secret for the service key, use the alias that you use
 +
 * `DISABLED_FOR_CONFIGS`: Disables the reporter for one or more configurations. The value is the name of one configuration or multiple names that are separated by commas. Names do not include the .dwl file extension.
 
-=== What to Do Next
+=== What To Do Next
```
Comment (hannanelson) [inline modules/ROOT/pages/bat-reporting-task.adoc]:
```suggestion
=== What to Do Next
```

File: modules/ROOT/pages/afm-create-private-location.adoc (hunk 3 above)
Comment (bjoson-mulesoft) [reply modules/ROOT/pages/afm-create-private-location.adoc]: Fixed all locally f/r

Comment (bjoson-mulesoft) [reply modules/ROOT/pages/afm-create-private-location.adoc]: Just pushed to remote

File: modules/ROOT/pages/bat-schedule-test-task.adoc [hunk 9]
```diff
@@ -1,12 +1,11 @@
...
-include::_attributes.adoc[]
-endif::[]
 
-This procedure describes how to schedule, run, and monitor an API test in Anypoint Platform using the default schedule and how to use a cron job to change the schedule. BAT is deployed by default in the Production environment of Anypoint Platform when you execute the bat login command. After deploying BAT, you execute a `bat schedule <options>` command. By default, if you don't include the optional cron expression in the command, the test runs every 15 minutes. The last step in this procedure changes the default to use a cron expression that runs the test once a day for a month.
+You can schedule, run, and monitor an API test in Anypoint Platform using the default schedule and use a cron job to change the schedule. BAT is deployed by default in the Production environment of Anypoint Platform when you execute the bat login command. After deploying BAT, you execute a `bat schedule <options>` command. By default, if you don't include the optional cron expression in the command, the test runs every 15 minutes. The last step in this procedure changes the default to use a cron expression that runs the test once a day for a month.
```
Comment (hannanelson) [inline modules/ROOT/pages/bat-schedule-test-task.adoc:3]: `bat login` command?

File: modules/ROOT/pages/afm-view-test-history.adoc (hunk 6 above)
Comment (bjoson-mulesoft) [reply modules/ROOT/pages/afm-view-test-history.adoc:1]: Hmm, I think History can be considered plural

File: modules/ROOT/pages/bat-start-new-project.adoc [hunk 10]
```diff
@@ -32,7 +29,7 @@ In this folder, you create configuration files to use for your tests. Configurat
 +
 For example, to refer to a variable named `url` in a configuration file, you use this reference: '$(config.url)'. When you run the `bat` command to execute a test, you specify the configuration file to use by including the `--config=file.dwl` option, where `file.dwl` is the path and name of the configuration file in the folder for your test suite.
 
-== What to Do Next
+== What To Do Next
```
Comment (hannanelson) [inline modules/ROOT/pages/bat-start-new-project.adoc]:
```suggestion
== What to Do Next
```

File: modules/ROOT/pages/afm-view-test-history.adoc (hunk 6 above)
Comment (bjoson-mulesoft) [reply modules/ROOT/pages/afm-view-test-history.adoc:1]: I could say "Viewing the History of Tests", but that seems overkill

**Review by hannanelson — APPROVED** [review]

**Pull request merged**